tests/components/netatmo/test_climate.py

```diff
@@ -36,8 +36,7 @@ async def test_webhook_event_handling_thermostats(hass, config_entry, netatmo_au
 
     assert hass.states.get(climate_entity_livingroom).state == "auto"
     assert (
-        hass.states.get(climate_entity_livingroom).attributes["preset_mode"]
-        == "Schedule"
+        hass.states.get(climate_entity_livingroom).attributes["preset_mode"] == "away"
```

Comment:
Is this changing because the fixture changed and not because it's a breaking change?

Reply:
Because the fixture data has changed.